src/components/HelpContent.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { getTranslationKeyForExplanation } from "@/utils/translationUtils";
+import { computed, type ComputedRef, type Ref } from "@vue/reactivity";
+import { inject } from "vue";
+import { useI18n } from "vue-i18n";
+import HelpCard from "./HelpCard.vue";
+import HelpModal from "./HelpModal.vue";
+
+// must use this if we want to use the translation function inside script setup
+const { t } = useI18n();
+
+const applicationGoalTitel: ComputedRef<string> = computed(() => {
```

Comment:
Computed Refs braucht man nur, wenn es 1) reactive sein soll und 2) wenn man irgendwas berechnen muss (deswegen auch computed). Hier trifft beides nicht zu, da es statische strings sind die du returnst. Es reicht also aus die direkt in das Template zu schreiben